src/commands/users/email-aliases/add.js

```diff
@@ -16,7 +23,11 @@ UsersAddEmailAliasCommand.aliases = [ 'users:add-email-alias' ];
 UsersAddEmailAliasCommand.description = 'Add a new email alias to a user';
 
 UsersAddEmailAliasCommand.flags = {
-	...BoxCommand.flags
+	...BoxCommand.flags,
+	confirm: flags.boolean({
+		allowNo: true,
+		description: 'Whether or not to confirm the email alias.  Only Admins may automatically confirm an alias.'
```

Comment:
Since this flag works by absence/presence instead of taking a value, it might be more accurate to say "Confirms the email alias automatically. Only Admins may automatically confirm an alias."

Reply:
@nidoran Since this flag has `allowNo: true`, the user could also pass `--no-confirm` to explicitly set the `is_confirmed` field to false.  Even though that's the default, I liked the idea of having a way for the user to specifically set it that way just in case that makes someone feel better.

Reply:
Thanks, I didn't look into that feature